cmake-modules/AddAzureSDK.cmake

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
+# How to use: Copy this file to a cmake modules folder within your project.
+# - Add `list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/your-cmake-modules-folder")`
+# - Call this module by doing `include(AddAzureSDK)` from root CMakeLists.txt
+# - Link your application against your cmake targets like `target_link_libraries(yourApp PRIVATE az_core)`
+#
+# This will create a _deps folder for the generated project and Azure SDK will be checked out in there.
+# When calling `cmake --build .`, Azure SDK will be built before your application and become available.
+#
+# Cmake options for Azure SDK for C can be set together with your application cmake options.
+#
+# Note: Update GIT_TAG to the expected version from Azure SDK for C
```

Comment:
Great comments! Nice work.

Reply:
thank you very much for the suggested changes. I am learning a lot from you how to write bettter